macros/edr/dbt_artifacts/upload_source_freshness.sql

```diff
@@ -22,16 +22,18 @@
       {% do execute_timing.update(timing) %}
     {% endif %}
   {% endfor %}
+  {% set metadata_dict = elementary.safe_get_with_default(node_dict, 'metadata', {}) %}
+  {% set source_freshness_invocation_id = metadata_dict.get('invocation_id', invocation_id) %}
   {% set flatten_source_freshness_dict = {
-       'source_freshness_execution_id': [invocation_id, node_dict.get('unique_id')] | join('.'),
+       'source_freshness_execution_id': [source_freshness_invocation_id, node_dict.get('unique_id')] | join('.'),
```

Comment:
Where is `source_freshness_invocation_id` defined?
What's the purpose of the change?